db/contacts.json

```diff
@@ -0,0 +1,94 @@
+{
+    "contacts": [
+        {
```

Comment:
This should be an object of objects with keys for each of the contacts (just like it will be from Firebase).